### ISSUE TITLE
 Chronograf (1.7.16) Expected number or array but got string: 100.

### ISSUE THREAD

**Issue by senyer** [influxdata/chronograf#5401]
What browser are you using (name and version)?

What operating system are you using?

Please describe what you were trying to do when you encountered this error:

```
Error: Expected number or array but got string: 100.
    at j (http://127.0.0.1:8888/src.a5ab7481.js:3173:1898)
    at W.parseArray_ (http://127.0.0.1:8888/src.a5ab7481.js:3173:31911)
    at W.start_ (http://127.0.0.1:8888/src.a5ab7481.js:3173:35574)
    at W.updateOptions (http://127.0.0.1:8888/src.a5ab7481.js:3173:36541)
    at d.value (http://127.0.0.1:8888/src.a5ab7481.js:3233:4903)
    at no (http://127.0.0.1:8888/src.a5ab7481.js:717:84207)
    at ao (http://127.0.0.1:8888/src.a5ab7481.js:717:85725)
    at http://127.0.0.1:8888/src.a5ab7481.js:717:98848
    at Object.exports.unstable_runWithPriority (http://127.0.0.1:8888/src.a5ab7481.js:713:3303)
    at Xo (http://127.0.0.1:8888/src.a5ab7481.js:717:98783)
```

**Comment by russorat** [comment]
@senyer thanks for the issue. could you try upgrading to 1.8 and see if the issue persists?

**Comment by sranka** [comment]
These kind of defects are already fixed (or better reported) with chronograf 1.9
